### PR TITLE
Refactor/css button functionality

### PR DESCRIPTION
**Pull request by ConnorAndersonLarson**
## Purpose
Refactors CSS/HTML to add button functionality for user input fields.

## Abstract
Input Fields for activities were added to the functionality of site, this PR integrates this functionality with the overall design of the site while also fixing some sizing issue with various info/calendar cards.

#### Pre-Merge TODOs
- [x] Check that user fields display properly
- [x] Check that cards display properly
- [x] Check Styling/Syntax
- [x] Check for opportunities for further refactoring

## Learning
The base construction of the HTML provided much trouble when setting return buttons. This is definitely not perfect; cards should be refactored to improve positioning of all items, but time is of the essence.

#### Next Steps
Improve site responsive, standardize return button placement (if time)

### CODE REVIEW

File: src/css/_variables.scss [hunk 1]
```diff
@@ -1,2 +1,8 @@
 $oswald: 'Oswald', sans-serif;
 $work: 'Work Sans', sans-serif;
+
+@mixin flex-row(){
```
Comment (owenhallgren) [inline src/css/_variables.scss:4]: I like this use of a mixin

**Review by owenhallgren — APPROVED** [review]

**Review by elizhahn — APPROVED** [review]

File: src/css/styles.scss [hunk 2]
```diff
@@ -161,6 +101,31 @@ main {
   border: 3px solid #1F1F1F;
 }
 
+// .info-card{
+//   height:100%;
+//
+// }
```
Comment (elizhahn) [inline src/css/styles.scss]: Delete this, unless you still need it?

File: src/css/styles.scss [hunk 3]
```diff
@@ -263,9 +231,22 @@ main {
...
     font-size: 5vw;
   }
+  h4{
+    font-size: 6vw;
+  }
   h5{
     font-size: 4vw;
   }
+  p{
+    //font-size: 4vw;
```
Comment (elizhahn) [inline src/css/styles.scss]: delete this?

**Review by elizhahn — APPROVED** [review]

**Comment by elizhahn** [comment]
I pulled down the branch and checked out the styling, When the display is full screen, the buttons are white, and when in mobile view, the buttons are colored. Was this on purpose?

File: src/css/styles.scss (hunk 3 above)
Comment (ConnorAndersonLarson) [reply src/css/styles.scss]: Done

File: src/css/styles.scss (hunk 2 above)
Comment (ConnorAndersonLarson) [reply src/css/styles.scss]: Good catch, done.

**Comment by ConnorAndersonLarson** [comment]
I did the color change because mobile doesn’t have hover effects. When you mouse over those buttons they change color to show
A.) It’s a button
B.) It lines up with the various cards
Since mobile doesn’t have hover options I made them a solid color to signify those things instead

**Comment by elizhahn** [comment]
I see the hover affects now, and how they change color when clicking on it. Nice work!

**Pull request merged**